src/pages/resource/ResourcePresentation.tsx

```diff
@@ -33,6 +35,17 @@ const PreviewComponentWrapper = styled.div`
   justify-content: center;
 `;
 
+const TransistorPreviewComponentWrapper = styled.div`
```

Comment:
https://blog.devgenius.io/using-styled-components-and-props-with-typescript-react-a3c32a496f47

Unngå px bruk, 16px = 1rem

Reply:
Takk!